src/shared/layouts/ProtectedNavBarLayout/ProtectedWrapper/ProtectedWrapper.tsx

```
@@ -0,0 +1,22 @@
+import { PropsWithChildren, useEffect } from 'react'
+
+import { ROUTES_URL } from '@/shared/const'
+import { loadFromLocalStorage } from '@/shared/lib/helpers'
+import { useRouter } from 'next/router'
+
+export const ProtectedWrapper = ({ children }: PropsWithChildren<{}>) => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)